_posts/2025-03-17-collecting-diagnostic-data.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "Collecting diagnostic data"
+date: 2025-03-17
+author: federico_valeri
+---
+
+Debugging distributed systems like Apache Kafka can be challenging.
+Without access to the right diagnostic data, identifying the root cause of an issue can become a time-consuming process.
+Strimzi, the Kubernetes-native Kafka operator, provides a [bash script](https://github.com/strimzi/strimzi-kafka-operator/blob/0.45.0/tools/report.sh) to collect cluster artifacts, including logs and configurations.
```

Comment:
As it is a not-released artifact, I wonder if linking to the main branch would be better. But it might be also worse if we one day decide to remove or move it. So up to you.

Reply:
I'm using the tag here exactly to avoid the risk of code changes.